README.md

```diff
@@ -20,18 +20,12 @@ We created this repo with the following goals in mind
 
 ## Install OAM runtime
 
-#### Clone this repo
-
-```console
-git clone git@github.com:crossplane/oam-kubernetes-runtime.git
-cd ./oam-kubernetes-runtime
-```
-
 #### Install OAM controllers
```

Comment:
How about removing this title [by updating title `Install OAM runtime`]?

Reply:
sure